code/modules/research/nanites/nanite_programs/protocols.dm

```diff
@@ -46,43 +46,25 @@
 	factory_efficiency = min(factory_efficiency + 1, max_efficiency)
 	nanites.adjust_nanites(null, round(0.002 * factory_efficiency, 0.1))
 
-/datum/nanite_program/protocol/tinker
-	name = "Tinker Protocol"
-	desc = "Replication Protocol: the nanites learn to use metallic material in the host's bloodstream to speed up the replication process."
+/datum/nanite_program/protocol/pyramid
```

Comment:
accidentally removing tinker protocol and replacing it with an unfinished protocol which just gives a straight bonus to nanite gain